r/NEWS.md

```diff
@@ -17,17 +17,23 @@
   under the License.
 -->
 
-# arrow 4.0.0.9000
+# arrow 4.0.0.9000 (under development)
```

Comment:
This line gets autogenerated/updated by a script so we shouldn't stick other information here

```suggestion
# arrow 4.0.0.9000
```

Reply:
Womp womp, that's unfortunate